src/main/java/com/facebook/openwifirrm/optimizers/tpc/TPC.java

```diff
@@ -84,25 +83,25 @@ public TPC(
 	}
 
 	/**
-	 * Update the tx power choices based on user and allowed channels from deviceConfig
+	 * Determine the new tx power choices based on user and allowed channels from deviceConfig.
+	 *
 	 * @param band the operational band
-	 * @param serialNumber the device
-	 * @param txPowerChoices the available tx powers of the device
-	 * @return the updated tx powers of the device
+	 * @param serialNumber the AP's serial number
+	 * @param txPowerChoices the AP's available tx powers
+	 * @return the AP's updated tx powers
```

Comment:
this was intentional, we refer to "device" in the comments because that is what the serial number represents (it's all over the codebase)

Reply:
We discussed - I'll keep it device for now. Maybe I'll update all references later though.